product-web-app/service/authService.ts

```diff
@@ -0,0 +1,21 @@
+import axios from 'axios';
+import dotenv from 'dotenv';
+import Employee from '../model/register.js';
+import registerValidator from '../validator/registrationValidator.js';
+
+dotenv.config();
+
+const BASE_URL = process.env.API_URL;
+
+async function register(employee: Employee): Promise<number> {
```

Comment:
Would be nice to have a class exported instead of a single function
```
export default class AuthService {
    (employee: Employee): Promise<number> {
}
```
You can apply it to other frontend services.

Reply:
done